docs/packages/working_with_packages.md

```diff
@@ -210,17 +219,20 @@ popd
 cp /tmp/gnuchess/gnuchess.spec .
 ```
 
-Now calculate the SHA-256 hashed for gnuchess-6.2.7.tar.gz and the book_1.01.pgn.gz file  The SHA-256 sum is used by the build system as an integrity check to ensure that the tarballs associated with a SPEC file are the expected one.
+### Create a Signature Meta-data File
```

Comment:
Do you think it's worth it to write a script for this part so that the user doesn't have to walk through the checksum steps?